src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -14,23 +17,40 @@ internal unsafe partial class Number
                 exponentBits: 11,
                 maxBinaryExponent: 1023,
                 exponentBias: 1023,
-                infinityBits: 0x7FF00000_00000000
+                infinityBits: 0x7FF00000_00000000,
+                smallerPowerOfTen: -342,
+                largerPowerOfTen: 308,
+                infinitePower: 0x7FF,
+                minExponentRoundToEven: -4,
+                maxExponentRoundToEven: 23,
+                maxExponentFastPath: 22
             );
 
             public static readonly FloatingPointInfo Single = new FloatingPointInfo(
                 denormalMantissaBits: 23,
                 exponentBits: 8,
                 maxBinaryExponent: 127,
                 exponentBias: 127,
-                infinityBits: 0x7F800000
+                infinityBits: 0x7F800000,
+                smallerPowerOfTen: -65,
```

Comment:
Same for this one, why is it `-65` and not `-45` due to: `~1.4012984643 * 10^-45`

Reply:
same.